### ISSUE TITLE
Better site availability during data import

### ISSUE THREAD

**Issue by sebbacon** [ebmdatalab/openprescribing#853]
Per [this comment](https://github.com/ebmdatalab/antibiotics-rct/issues/55#issuecomment-387757995), we get:

* Inconsistent data during imports (e.g. some measures will have the most recent month's data, some won't)
* Duff data following imports (e.g. when the data format has changed, etc)

The expedient fix is a site banner during import. Something like:  "We are currently importing data for <month> <year>. Until <time>, the site may be slower than normal, and you may see inconsistent data for the most recent month."

The "correct" fix is a staging database to which we write the latest data.  This is then reviewed, and then the relevant tables are copied across atomically following QA (preferably at night time).

The problem with the "correct" fix is cost: we'd at least need to spend an extra $500 / month (from memory) on storage.  If we were to do this, I'd be tempted to separate out our DB server from the OP web server at the same time.

**Comment by HelenCEBM** [comment]
The expedient fix (banner) is implemented - do we plan to do the complete fix at any point?

**Comment by sebbacon** [comment]
Yes this should be reviewed when the matrix data store is complete. Adding a "matrix" label for such things. cc @inglesp @evansd
